frontend/components/Navbar/Navbar.tsx

```diff
@@ -1,16 +1,14 @@
-import { getEmail, getUtorid } from '@/lib/frontendUtils';
-import { Persona } from '@/components/fluent';
-import Link from 'next/link';
+import { Link, Persona } from '@fluentui/react-components';
 import styles from './Navbar.module.css';
-import { GenerateInitalsAvatarProps } from '../server';
+import { GenerateInitalsAvatarProps } from '@/components';
 
 /**
  * The navbar component is the top bar of the website. It contains the
  * logo and the user's name and role.
  * @returns {JSX.Element} the navbar
  */
 export const Navbar = (): JSX.Element => {
-    const user = { utorid: getUtorid(), email: getEmail() };
+    const user = { utorid: "michael", email: "liut" };
```

Comment:
Assuming this is mock data for now, should add a central function like `getUser` so it is easier to integrate in the future